app/src/main/java/com/kickstarter/libs/AnalyticEvents.kt

```diff
@@ -272,6 +273,28 @@ class AnalyticEvents(trackingClients: List<TrackingClientType?>) {
         client.track("Application Login or Signup", properties)
     }
 
+    /**
+     * Tracks a login or sign up button clicked.
+     * @param type
+     * @param page
+     */
+    fun trackLoginOrSignUpCtaClicked(type: String?, page: String) {
+        val props: HashMap<String, Any> = hashMapOf(CONTEXT_CTA.contextName to LOGIN_OR_SIGN_UP.contextName)
+        props[CONTEXT_PAGE.contextName] = page
+        type?. let { props[CONTEXT_TYPE.contextName] = it }
```

Comment:
Do we need an overloaded method for this? We can use the same method and pass in null for `type` when it is not needed.

Reply:
yeah, correct since I have made the parameter optional. thanks